lib/node_modules/@stdlib/_tools/scaffold/math-special-unary/data/docs/repl__txt.txt

```diff
@@ -0,0 +1,67 @@
+
+{{alias}}( x[, options] )
+    {{REPL_TEXT_MAIN_DESC}}
+
+    The function returns an ndarray having the same shape as the input ndarray.
+
+    Parameters
+    ----------
+    x: ndarray
+        Input array. Must have a numeric or "generic" data type.
```

Comment:
@gunjjoshi Looks like we need an "input policy phrase" here. "numeric" is not true for all apis (e.g., those not supporting complex input dtypes).